README.md

```diff
@@ -298,32 +298,37 @@ This setup is heavyweight and matches a realworld deployment. It is useful to re
 
 Download the recommended version of minikube and kubectl :
 
-| Software | Version recommendation | Last Version | Link                                                                                                  |
-|----------|------------------------|--------------|-------------------------------------------------------------------------------------------------------|
-| kubectl  | 1.21+                  | 1.24.X       | [Download](https://storage.googleapis.com/kubernetes-release/release/v1.24.3/bin/linux/amd64/kubectl) |
-| minikube | 1.21+                  | 1.26.X       | [Download](https://storage.googleapis.com/minikube/releases/latest/minikube-linux-amd64)              |
+| Software | Version recommendation | Last supported version | Link                                                                                                  |
+|----------|------------------------|------------------------|-------------------------------------------------------------------------------------------------------|
+| kubectl  | 1.21+                  | 1.27.4                 | [Download](https://storage.googleapis.com/kubernetes-release/release/v1.27.4/bin/linux/amd64/kubectl) |
+| minikube | 1.21+                  | 1.31.2                 | [Download](https://storage.googleapis.com/minikube/releases/v1.31.2/minikube-linux-amd64)             |
 
 
 install [minikube](https://kubernetes.io/fr/docs/tasks/tools/install-minikube/#installez-minikube-par-t%C3%A9l%C3%A9chargement-direct) and [kubectl](https://kubernetes.io/fr/docs/tasks/tools/install-kubectl/#installer-le-binaire-de-kubectl-avec-curl-sur-linux) following instructions for binaries download installation.
 
 __Notes__: We require minikube 1.21+ for host.minikube.internal support inside containers (if you want to use an older minikube, replace host.minikube.internal with the IP of your host).
 
-Start minikube and activate ingress support:
+__Notes__: Minikube 1.32.0 has been tested and is not working on our stack, so please use version 1.31.2 or below.
```

Comment:
Strange because I have this version and it worked last time?
```shell
$ minikube version
minikube version: v1.32.0
commit: 8220a6eb95f0a4d75f7f2d7b14cef975f050512d
```

Reply:
I'm not sure how this could have worked with v1.32.0. v1.32.0 uses ingress-nginx/controller image 1.9.4 (https://github.com/kubernetes/minikube/releases/tag/v1.32.0). ingress-nginx/controller version >=1.9 introduced a breaking change (https://github.com/kubernetes/ingress-nginx/pull/10393) to fix a CVE (https://github.com/kubernetes/ingress-nginx/issues/10571). We should check how we want to handle that in our local deployment because we use configuration-snippet which is handled differently now https://github.com/gridsuite/deployment/blob/a95ea56d3940cad5c053d0c5c38a7e2a650735a1/k8s/live/local/balances-adjustment-server-ingress.yaml#L6-L26 
Do we still need this? We don't use springfox anymore? Use `allowSnippetAnnotations: true`?

With version > 1.9 of ingress-nginx/controller the validation error when deploying is:
`Error from server (BadRequest): error when creating "k8s/live/local": admission webhook "[validate.nginx.ingress.kubernetes.io](http://validate.nginx.ingress.kubernetes.io/)" denied the request: [nginx.ingress.kubernetes.io/configuration-snippet](http://nginx.ingress.kubernetes.io/configuration-snippet) annotation cannot be used. Snippet directives are disabled by the Ingress administrator`

Reply:
@antoinebhs : I created a ticket for this point.